tox.ini

```diff
@@ -221,6 +221,7 @@ commands = python -m pytest tests/integ/modin/test_old_numpy_aliases.py
 
 [testenv:datasource]
 description = run the datasource tests that requires DBMS drivers
+install_command = python -I -m pip install --only-binary psycopg2-binary {opts} {packages}
 deps =
     {[testenv]deps}
     databricks-sql-connector
```

Comment:
seems like test failed because psycopg2 is not added here

Reply:
I installed `pip install --only-binary psycopg2-binary` through the `install_command`
pip tries to install from source code for macos on github action node which would fail, I don't understand why..